docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb

```diff
@@ -0,0 +1,1316 @@
+       "      <td>23.59</td>\n",
+       "      <td>15811197</td>\n",
+       "      <td>182</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>1754155 rows × 9 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         TRANSACTION_ID         TX_DATETIME CUSTOMER_ID TERMINAL_ID  \\\n",
+       "0                     0 2018-04-01 00:00:31         596        3156   \n",
+       "1                     1 2018-04-01 00:02:10        4961        3412   \n",
+       "2                     2 2018-04-01 00:07:56           2        1365   \n",
+       "3                     3 2018-04-01 00:09:29        4128        8737   \n",
+       "4                     4 2018-04-01 00:10:34         927        9906   \n",
+       "...                 ...                 ...         ...         ...   \n",
+       "1754150         1754150 2018-09-30 23:56:36         161         655   \n",
+       "1754151         1754151 2018-09-30 23:57:38        4342        6181   \n",
+       "1754152         1754152 2018-09-30 23:58:21         618        1502   \n",
+       "1754153         1754153 2018-09-30 23:59:52        4056        3067   \n",
+       "1754154         1754154 2018-09-30 23:59:57        3542        9849   \n",
+       "\n",
+       "         TX_AMOUNT TX_TIME_SECONDS TX_TIME_DAYS  TX_FRAUD  TX_FRAUD_SCENARIO  \n",
+       "0            57.16              31            0         0                  0  \n",
+       "1            81.51             130            0         0                  0  \n",
+       "2           146.00             476            0         0                  0  \n",
+       "3            64.49             569            0         0                  0  \n",
+       "4            50.99             634            0         0                  0  \n",
+       "...            ...             ...          ...       ...                ...  \n",
+       "1754150      54.24        15810996          182         0                  0  \n",
+       "1754151       1.23        15811058          182         0                  0  \n",
+       "1754152       6.62        15811101          182         0                  0  \n",
+       "1754153      55.40        15811192          182         0                  0  \n",
+       "1754154      23.59        15811197          182         0                  0  \n",
+       "\n",
+       "[1754155 rows x 9 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def load_date(filename):\n",
+    "    print(\".\",end=\"\", flush=True)\n",
+    "    return pd.read_pickle(f\"https://github.com/Fraud-Detection-Handbook/simulated-data-raw/raw/main/data/{filename}.pkl\")\n",
+    "\n",
+    "print(\"Downloading dataset\",end=\"\")\n",
+    "with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:\n",
```

Comment:
Is this really necessary?

Reply:
Otherwise, the dataset takes a long time to load. Increase the number of threads to 4 (I got throttled by github during development).